transaction-flow.md

```diff
@@ -697,7 +708,7 @@ This is the amount that has to be collected from the sender, or funded from the
 
 ### sender
 
-The full details of the sender. If this is a new sender, please note of the `id` field, as that MUST be used in subsequent transaction creation calls that are from the same sender
+The full details of the sender. If this is a new sender, please make a note of the `id` field, as that MUST be used in subsequent transaction creation calls that are from the same sender
```

Comment:
Add details that the `external_id` can also be used to link senders. We might need to include more details on the available options, e.g.

1. Only and `id` is sent - we search for this sender and use it
2. Only an `external_id` is sent - we search for this sender and use it
3. An `external_id` is sent along with additional fields - we create a new sender - throwing an error if the external_id is already used - and return the matching duplicate